README.md

```diff
@@ -613,6 +613,44 @@ app.get(layout.pathname(), (req, res) => {
 });
 ```
 
+### res.podiumStream(...templateArguments)
+
+Method on the `http.ServerResponse` object for streaming HTML to the browser. This function returns a `ResponseStream` object that can then be used to push out HTML to the browser in chunks using its .send() function. Once streaming is finished, the .done() function must be called to close the stream.
+
+```js
+const stream = res.podiumStream();
+stream.send(`<div>HTML chunk 1</div>`);
+stream.send(`<div>HTML chunk 2</div>`);
+stream.done();
+```
+
+The Podium document template will still be used. When you call res.podiumStream(), the document head will be sent to the browser immediately. Once the .done() function is called, the closing part of the template will be sent before the stream is closed out.
+
+Note that any arguments passed to .podiumStream(...args) will be passed on to the layout's document template.
```

Comment:
I wonder if this should be an options object so we can add to it without causing a breaking change.

```
res.podiumStream({
  view: {
    templateArgs: [...args],
  },
});
```

Reply:
I don't think there are any breaking changes to worry about here. Did you spot something?

Reply:
Not now, I'm thinking if we want to add stuff in the future.